scripts/compare-dumps.go

```diff
@@ -8,10 +8,13 @@ import (
 	"os"
 	"path/filepath"
 	"sort"
+	"strings"
 
 	"github.com/urfave/cli"
 )
 
+const ledgerSIPrefix string = "/v///w"
```

Comment:
Really non-trivial and not really correct. You can only compare base64-decoded values for `feffffff` prefix, because `/v///` prefix is guaranteed, but the 6th character is not, it only has 2 out of 6 bits from the original 32-bit contract ID:
```
$ printf \\xfe\\xff\\xff\\xff | base64
/v///w==
$ printf \\xfe\\xff\\xff\\xff\\0 | base64
/v///wA=
$ printf \\xfe\\xff\\xff\\xff\\xff | base64
/v////8=
```

Reply:
Nice example, fixed.